internal/alpini/router/alpini-router-base/src/test/java/com/linkedin/alpini/router/TestScatterGatherRequestHandlerImpl.java

```diff
@@ -86,14 +87,35 @@ private interface Response {
     void setKeepAlive(boolean keepAlive);
   }
 
-  private enum Status {
-    OK(200), MULTI_STATUS(207), BAD_REQUEST(400), NOT_FOUND(404), TOO_BUSY(429), INTERNAL_SERVER_ERROR(500),
-    GATEWAY_TIMEOUT(502), SERVICE_UNAVAILABLE(503);
+  private static class Status extends HttpResponseStatus {
+    static final Status OK = new Status(200, "OK");
```

Comment:
Why this change?

Reply:
This change is to make the test-only class `Status` to be compatible with `HttpResponseStatus`. Without this, the casting from HRS to `HttpResponseStatus` in [this line](https://github.com/linkedin/venice/blob/ca18d4f7a3fd9bc51128a038b810a9e6860d6034/internal/alpini/router/alpini-router-base/src/main/java/com/linkedin/alpini/router/ScatterGatherRequestHandlerImpl.java#L682) would fail in tests because `Status` didn't respect it.